components/supabase/supabase.app.mjs

```diff
@@ -31,6 +31,17 @@ export default {
       description: "Filter type for the query",
       options: constants.FILTER_OPTIONS,
     },
+    sortOrder: {
+      type: "string",
+      label: "Sort Order",
+      description: "Sort ascending or descending",
```

Comment:
```suggestion
      description: "Sort ascending or descending.",
```

Reply:
Added a period at the end of the description to follow Google's Developer docs style guide.